Makefile

```diff
@@ -112,7 +101,7 @@ all-srcs:
 cover: nocover
 	@echo pre-compiling tests
 	@time go test -i $(ALL_PKGS)
-	@./scripts/cover.sh $(shell go list $(TOP_PKGS))
+	@./scripts/cover.sh $(shell go list ./...)
```

Comment:
I think we need to get rid of this script, it's no longer needed. See https://github.com/opentracing/opentracing-go/pull/202

But can do in another PR (add a TODO)